palette/benchmarkgo17_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build go1.7
+
+package palette
+
+import (
+	"fmt"
+	"image/color"
+	"math/rand"
+	"testing"
+)
+
+func BenchmarkColorMap_At(b *testing.B) {
```

Comment:
Well, the obvious answer is if they are informative we should keep them, right? If by "informative" you meant "not informative", then we probably shouldn't keep them, however.

My initial thinking was that this method could theoretically end up in inner loops, so it might be useful to have a benchmark.